src/main/java/fi/tkgwf/ruuvi/handler/BeaconHandler.java

```diff
@@ -28,9 +28,6 @@
         if (adData == null) {
             adData = hciData.findAdvertisementDataByType(0x16); // Eddystone url
         }
-        if (adData == null || !Config.isAllowedMAC(hciData.mac)) {

```

Comment:
the nullcheck is still necessary here in case a packet is being parsed that is neither manufacturer specific data nor eddystone